roles/openshift_version/tasks/main.yml

```diff
@@ -5,6 +5,12 @@
     is_containerized: "{{ openshift.common.is_containerized | default(False) | bool }}"
     is_atomic: "{{ openshift.common.is_atomic | default(False) | bool }}"
 
+- name: Install the base package for versioning
```

Comment:
We're installing the base package on all hosts which will require subs on some hosts we didn't require subs on before such as loadbalancer hosts.